### ISSUE TITLE
RecordTitle is not importable like the rest

### ISSUE THREAD

**Issue by christiaanwesterbeek** [marmelab/react-admin#2018]
**What you were expecting:**
RecordTitle to be importable intuitively.

**What happened instead:**
it's not importable intuitively.

**Steps to reproduce:**
`import { RecordTitle } from 'react-admin';`

RecordTitle is undefined.
_Update: I wrote "is not undefined" by accident. Fixed it now_

Of course I could:
`import RecordTitle from 'ra-ui-materialui/lib/layout/RecordTitle';`

**Related code:**
Of all files in https://github.com/marmelab/react-admin/tree/master/packages/ra-ui-materialui/src/layout everything is exported in `/packages/ra-ui-materialui/src/layout/index.js` except for RecordTitle

```
export AppBar from './AppBar';
export AppBarMobile from './AppBarMobile';
export CardActions from './CardActions';
export Confirm from './Confirm';
export DashboardMenuItem from './DashboardMenuItem';
export Header from './Header';
export Layout from './Layout';
export Loading from './Loading';
export LinearProgress from './LinearProgress';
export LoadingIndicator from './LoadingIndicator';
export Menu from './Menu';
export MenuItemLink from './MenuItemLink';
export NotFound from './NotFound';
export Notification from './Notification';
export RecordTitle from './RecordTitle';  // <-- Missing or at least I expected it here.
export Responsive from './Responsive';
export Sidebar from './Sidebar';
export Title from './Title';
export ViewTitle from './ViewTitle';
```

**Environment**

* React-admin version: 2.1.1

**Comment by christiaanwesterbeek** [comment]
I notices something similar for `CreateActions`. It's not intuitively importable like `EditActions` and `ShowActions` are.

https://github.com/marmelab/react-admin/blob/master/packages/ra-ui-materialui/src/detail/index.js

For now, I'm importing them both like this:
```javascript
import RecordTitle from 'ra-ui-materialui/lib/layout/RecordTitle';
import DefaultActions from 'ra-ui-materialui/lib/detail/CreateActions';
```

PS: I know these are minor issues...

**Comment by fzaninotto** [comment]
Fixed by #2049
